client/src/app.tsx

```diff
@@ -84,18 +93,6 @@ function PageOrPageNotFound({ pageNotFound, children }) {
   );
 }
 
-function LoadingFallback({ message }: { message?: string }) {
```

Comment:
Q: Was this fallback part of the problem? If not, did you intentionally remove it, or could we still add it to the `React.Suspense` in `LazyStandardLayout`?

Reply:
Yes. The `LazyStandardLayout` wasn't there initially.